src/main/java/de/cronn/jira/sync/domain/JiraRemoteLink.java

```diff
@@ -6,7 +6,7 @@
 import org.apache.commons.lang3.builder.ToStringBuilder;
 import org.apache.commons.lang3.builder.ToStringStyle;
 
-public class JiraRemoteLink extends JiraIdResource {
+public class JiraRemoteLink {
```

Comment:
Why is this change good/required? How is it related to the component change?

Reply:
It doesn't seem to be required. I must have changed it accidentally.

I reverted the change.